TodoMate/Todo.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
웹 접근성을 위해서 바꿔주는게 좋대요

- 스크린리더 작동 시 한국어로 읽어줌
- 크롬 자동번역 인터페이스 사용시 언어 감지

[[HTML] lang 속성에 대하여
](https://velog.io/@ww8007/HTML-lang-%EC%86%8D%EC%84%B1%EC%97%90-%EB%8C%80%ED%95%98%EC%97%AC)

(나 kr이랑 ko 헷갈려서 kr 적음ㅋㅋ korean은 ko 라고 합디다...)
```suggestion
<html lang="ko">
```